lib/OpenQA/Schema/Result/Jobs.pm

```diff
@@ -948,6 +958,11 @@ sub update_status {
 
     if ($self->worker) {
         $self->worker->set_property("INTERACTIVE", $status->{status}->{interactive} // 0);
+        # mark the worker as alive
+        $self->worker->seen;
```

Comment:
why add here? Now I can not ensure anymore by review that this is just a simple "add log in case of error" but could change functionality